src/main/java/gregtech/common/metatileentities/steam/multiblockpart/MetaTileEntitySteamItemBus.java

```diff
@@ -0,0 +1,72 @@
+package gregtech.common.metatileentities.steam.multiblockpart;
+
+import codechicken.lib.render.CCRenderState;
+import codechicken.lib.render.pipeline.IVertexOperation;
+import codechicken.lib.vec.Matrix4;
+import gregtech.api.metatileentity.MetaTileEntity;
+import gregtech.api.metatileentity.MetaTileEntityHolder;
+import gregtech.api.metatileentity.multiblock.IMultiblockAbilityPart;
+import gregtech.api.metatileentity.multiblock.MultiblockAbility;
+import gregtech.api.metatileentity.multiblock.MultiblockControllerBase;
+import gregtech.api.render.ICubeRenderer;
+import gregtech.api.render.SimpleOverlayRenderer;
+import gregtech.api.render.Textures;
+import gregtech.common.ConfigHolder;
+import gregtech.common.metatileentities.electric.multiblockpart.MetaTileEntityItemBus;
+import net.minecraft.client.resources.I18n;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.world.World;
+import net.minecraftforge.items.IItemHandlerModifiable;
+
+import javax.annotation.Nullable;
+import java.util.List;
+
+public class MetaTileEntitySteamItemBus extends MetaTileEntityItemBus implements IMultiblockAbilityPart<IItemHandlerModifiable> {
+
+    private final boolean isExportHatch;
+
+    public MetaTileEntitySteamItemBus(ResourceLocation metaTileEntityId, boolean isExportHatch) {
+        super(metaTileEntityId, 1, isExportHatch);
+        this.isExportHatch = isExportHatch;
```

Comment:
Please add `this.setPaintingColor(0xFFFFFF)`. It prevents tinting of this hatch and causing multiblocks to look slightly wrong if the feature is utilized. Also, why is initializeInventory() not called here as well?